src/main/webapp/app/entities/file-upload-exercise/file-upload-exercise.route.ts

```diff
@@ -34,7 +34,7 @@ export class FileUploadExerciseResolve implements Resolve<FileUploadExercise> {
                 filter(res => !!res.body),
                 map((course: HttpResponse<Course>) => {
                     const fileUploadExercise = new FileUploadExercise(course.body!);
-                    fileUploadExercise.filePattern = 'pdf, png';
+                    fileUploadExercise.filePattern = '.pdf, .png';
```

Comment:
Is it necessary to set a hard-coded default here for the file pattern? If not, please remove it. Also, it would be better to provide a list of file patters instead of a string.

Reply:
Also I wonder if we should add the dot here, it is unnecessary from a user's point of view.

Reply:
@lovaiible I would leave the hard-coded file pattern, so the user has a better idea how to enter it. Regarding the list of file patterns, this would be best implemented if we have selection of file extensions. In current state we would have to store it as a list on the server, but then handle it as a `coma-separated` string on the client.

Reply:
@thilo-behnke I think it makes more sense with a dot, this is how you usually know that it is a file extension and not a file name. But maybe I am looking on this from a different perspective

Reply:
To the user it should be clear that a file ending has a dot (when would it not?). This should just be something that is validated in the form.